### PR TITLE
issue 29714, add JULIA_DEPOT_PATH

### PR DESCRIPTION
**Pull request by mikephelan**
add JULIA_DEPOT_PATH

### CODE REVIEW

File: doc/src/manual/environment-variables.md [hunk 1]
```diff
@@ -107,6 +107,16 @@ Suppose the value of `$JULIA_PKGRESOLVE_ACCURACY` is `n`. Then
 *   the number of iterations between decimation steps is `10*n`, and
 *   at decimation steps, at most one in every `20*n` packages is decimated.
 
+### `JULIA_DEPOT_HOME`
+
+A directory path that points to the default location for storage of Julia packages. If `$JULIA_DEPOT_HOME` is not set, then `REPL.DEPOT_PATH` defaults to
+
+```
+$HOME/.julia
+$HOME/.julia/Julia-<release>/local/share/julia
```
Comment (vchuravy) [inline doc/src/manual/environment-variables.md]: `julia` and not `Julia`.  And maybe add that <release> is `v$major.$minor`

File: doc/src/manual/environment-variables.md [hunk 2]
```diff
@@ -107,6 +107,16 @@ Suppose the value of `$JULIA_PKGRESOLVE_ACCURACY` is `n`. Then
 *   the number of iterations between decimation steps is `10*n`, and
 *   at decimation steps, at most one in every `20*n` packages is decimated.
 
+### `JULIA_DEPOT_HOME`
```
Comment (fredrikekre) [inline doc/src/manual/environment-variables.md]: `JULIA_DEPOT_HOME` -> `JULIA_DEPOT_PATH`

File: doc/src/manual/environment-variables.md [hunk 3]
```diff
@@ -107,6 +107,16 @@ Suppose the value of `$JULIA_PKGRESOLVE_ACCURACY` is `n`. Then
...
+
+A directory path that points to the default location for storage of Julia packages. If `$JULIA_DEPOT_HOME` is not set, then `REPL.DEPOT_PATH` defaults to
```
Comment (fredrikekre) [inline doc/src/manual/environment-variables.md]: `JULIA_DEPOT_HOME` -> `JULIA_DEPOT_PATH` and `REPL.DEPOT_PATH` -> `Base.DEPOT_PATH`

**Comment by laborg** [comment]
`JULIA_DEPOT_PATH` has been documented in https://github.com/JuliaLang/julia/pull/30959 and in https://github.com/JuliaLang/julia/pull/31009

**Pull request closed without merging**